components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/gen/java/org/wso2/carbon/identity/api/server/api/resource/v1/ApiResourceCreationModel.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
```

Comment:
Update the license

Reply:
Please check the other places as well

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/473/commits/907f522f5b71eb350e9aa926b5ba3580f0022746